sql/moz-fx-data-shared-prod/telemetry_derived/hcm_clients_v1/query.sql

```diff
@@ -0,0 +1,23 @@
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  normalized_channel AS channel,
+  environment.system.os.name AS os,
+  COUNT(DISTINCT client_id) AS num_clients,
+FROM
+  `moz-fx-data-shared-prod`.telemetry_stable.main_v5
+CROSS JOIN
+  UNNEST(payload.processes.parent.keyed_scalars.a11y_theme)
+WHERE
+  {% if is_init() %}
+    DATE(submission_timestamp) >= '2021-01-01'
+  {% else %}
+    DATE(submission_timestamp) = @submission_date
+  {% endif %}
+  AND payload.processes.parent.keyed_scalars.a11y_theme IS NOT NULL
+  AND ARRAY_LENGTH(payload.processes.parent.keyed_scalars.a11y_theme) > 0
+  AND value IS TRUE
+  AND key != "always"
```

Comment:
Some description of this business logic, either here or in the table description, could be useful

Reply:
@MReschenberg could you provide a description for this logic

Reply:
"value" is derived from a LookAndFeel boolean which is TRUE when an OS high contrast mode (HCM) is present and FALSE otherwise. OS HCM is defined as HCM on windows, Increase contrast on macOS, and high contrast theme on linux. 

"key" is derived from the config pref `browser.display.document_color_use" which is has the following values / meanings: 
- "always" = always use document colors, regardless of OS HCM or colors chosen in the colors dialog (about:preferences > manage colors)
- "never" = never use document colors (this is equivalent to "always use HCM colors")
- "default" = use document colors when OS HCM is off, and use OS HCM colors when OS HCM is on.

Reply:
In the context of the entire query: we're looking for users who have OS HCM on and who have _not_ forced document colors everywhere (ie. the OS HCM colors should propagate to web content).

Reply:
thank you!